drivers/kscan/kscan_ite_it8xxx2.c

```diff
@@ -192,11 +200,16 @@ static bool read_keyboard_matrix(const struct device *dev, uint8_t *new_state)
 static void keyboard_raw_interrupt(const struct device *dev)
 {
 	const struct kscan_it8xxx2_config *const config = DRV_CONFIG(dev);
-	volatile uint8_t *reg_wuesr3 = (uint8_t *)config->reg_wuesr3;
 	struct kscan_it8xxx2_data *data = DRV_DATA(dev);
 
-	/* W/C wakeup interrupt status of KSI[0:7] pins */
-	*reg_wuesr3 = 0xFF;
+	/*
+	 * W/C wakeup interrupt status of KSI[7:0] pins
+	 *
+	 * NOTE: We want to clear the status as soon as possible,
+	 *       so clear KSI[7:0] pins at a time.
+	 */
+	it8xxx2_wuc_clear_status(config->wuc_map_list[0].group,
```

Comment:
This call becomes `it8xxx2_wuc_clear_status(config->wuc_map_list[0].wucs, KEYBOARD_KSI_PIN_MASK);` after changing the `it8xxx2_wuc_clear_status` prototype.

Reply:
Done